cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -131,14 +143,6 @@
   <!-- Router Configuration -->
   
   <property>
-    <name>router.address</name>
```

Comment:
You shouldn't remove things that are in use *prior* to discussion. If you want to remove this, fix the scripts, also. Otherwise, revert this change. Don't break things, please.